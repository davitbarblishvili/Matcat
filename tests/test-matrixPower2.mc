func main() int {
    matrix a;
    matrix b;
    a = [[1,2,2],[4,5,6],[7,0,9]];
    b = a^-3;

    printm(b);

    return 0;

}