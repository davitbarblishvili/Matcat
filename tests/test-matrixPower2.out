(
[-35.18 13.25 -0.99 ]
[-3.40 1.29 -0.11 ]
[26.45 -9.97 0.75 ]
)